server/handler/tables.go

```diff
@@ -2,18 +2,58 @@ package handler
 
 import (
 	"net/http"
-	"strings"
 
 	"github.com/src-d/gitbase-playground/server/serializer"
 	"github.com/src-d/gitbase-playground/server/service"
 )
 
-// Tables returns a function that calls /query with the SQL "SHOW TABLES"
+// Tables returns DB schema
 func Tables(db service.SQLDB) RequestProcessFunc {
-	return func(r *http.Request) (*serializer.Response, error) {
-		req, _ := http.NewRequest("POST", "/query",
-			strings.NewReader(`{ "query": "SHOW TABLES" }`))
+	return func(r *http.Request) (res *serializer.Response, err error) {
```

Comment:
I'd move this logic somewhere and I'd test it.
simple test cases, nothing complex:
- database with a couple of tables with a couple of cols,
- an empty database,
- an error,

Reply:
we don't have such infrastructure for tests. And implementing it is definitely out of the scope of this PR.